lightkurve/tests/test_synthetic_data.py

```diff
@@ -0,0 +1,188 @@
+from __future__ import division, print_function
+
+import numpy as np
+
+from astropy.utils.data import get_pkg_data_filename
+import pytest
+import warnings
+
+from ..targetpixelfile import KeplerTargetPixelFile, TessTargetPixelFile
+from ..correctors import SFFCorrector, PLDCorrector
+
+filename_synthetic_sine = get_pkg_data_filename("data/synthetic/synthetic-k2-sinusoid.targ.fits.gz")
+filename_synthetic_transit = get_pkg_data_filename("data/synthetic/synthetic-k2-planet.targ.fits.gz")
+filename_synthetic_flat = get_pkg_data_filename("data/synthetic/synthetic-k2-flat.targ.fits.gz")
+
+
+lacks_BLS = False
```

Comment:
This isn't a big deal, but PEP8 convention is to use `lower_case_with_underscores` for all variable names.